doc/main.html

```diff
@@ -1622,6 +1655,259 @@ <h3>Complex Example</h3>
+
+      </section>
+
+    </section>
+    <section id="sec-list-nesting">
+      <h3>List Nesting</h3>
+
+      <p>
+        Both <code>ol</code> and <code>ul</code> list may be nexted as many times within a <code>li</code> element as needed to achieve the required results. See below for various examples. 
+      </p>
+
+      <div class="example">
+        <p>
+          Nesting <code>ol</code>:
+        </p> 
+        <pre>
+&lt;ol&gt;
+  &lt;li&gt;First list item
+    &lt;ol&gt;
+      &lt;li&gt;First sub item&lt;/li&gt;
+      &lt;li&gt;Second sub item&lt;/li&gt;
+    &lt;/ol&gt;
+  &lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ol&gt;</pre>
+        
+        <p>is rendered as:</p>
+        <ol>
+          <li>First list item
+            <ol>
+              <li>First sub item</li>
+              <li>Second sub item</li>
+            </ol>
+          </li>
+          <li>Second list item</li>
+        </ol>
+      </div>
+
+      <div class="example">
+        <p>
+          Nesting <code>ul</code>:
+        </p> 
+        <pre>
+&lt;ul&gt;
+  &lt;li&gt;First list item
+    &lt;ul&gt;
+      &lt;li&gt;First sub item&lt;/li&gt;
+      &lt;li&gt;Second sub item&lt;/li&gt;
+    &lt;/ul&gt;
+  &lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ul&gt;</pre>
+        
+        <p>is rendered as:</p>
+        <ul>
+          <li>First list item
+            <ul>
+              <li>First sub item</li>
+              <li>Second sub item</li>
+            </ul>
+          </li>
+          <li>Second list item</li>
+        </ul>
+      </div>
+
+      <div class="example">
+        <p>
+          Nesting <code>ul</code> in <code>ol</code>:
+        </p> 
+        <pre>
+  &lt;ol&gt;
+  &lt;li&gt;First list item
+    &lt;ul&gt;
+      &lt;li&gt;First sub item&lt;/li&gt;
+      &lt;li&gt;Second sub item&lt;/li&gt;
+    &lt;/ul&gt;
+  &lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+  &lt;/ol&gt;</pre>
+        
+        <p>is rendered as:</p>
+        <ol>
+          <li>First list item
+            <ul>
+              <li>First sub item</li>
+              <li>Second sub item</li>
+            </ul>
+          </li>
+          <li>Second list item</li>
+        </ol>
+      </div>
+
+    </section>
+    <section id="sec-list-complex-example">
+      <h3>Complex List Example</h3>
+
+      <p>
+        The features in <a href="#sec-ordered-lists"></a> and <a href="#sec-list-nesting"></a> may be combined as needed to achieve the required results for a complex list. See below for an example. 
+      </p>
+
+      <div class="example">
+        <p>
+          Nesting <code>ul</code> in <code>ol</code>:
+        </p> 
+        <pre data-include="snippets/list-sample-complex.txt"></pre>
+        
+        <p>is rendered as:</p>
+        <ol>
+          <li>First list item
+            <ul>
+              <li>First sub item
+                <ol type="i">
+                  <li>First sub item's 1st item with small roman numerals</li>
```

Comment:
I am not sure I am in love with sub-bullets containing numbered items. How do were cite/reference that?

Reply:
Do we NEED to be able to reference items in a list? How would we do this for a `<ul>` in general, and why would this be different than that? That being said, I can't find any instances of mixed `ol` and `ul` in any docs so far. Maybe we prevent the mix of the two in general, and I also found this in the ISO guidelines for lists (we may need to add allowance to references for `ol`):
```
23.4 Referencing
The purpose of a list should be made clear by its context. For example, an introductory proposition or a subclause title can serve to introduce the list.

If cross-references to list items are necessary, a numbered list shall be used. Within a subdivision, each list item in a numbered list shall have a unique identifier. Numbering restarts at each new clause or subclause.

Use, for example, the following forms for references to lists:

• “as specified in 3.1 b)”;

• “the requirements given in Clause B.2 c)”.
```

Reply:
I do not think we should prohibit mixing using validation, but perhaps show examples where `ol` is not embedded in `ul`

Reply:
Done in https://github.com/SMPTE/html-pub/pull/155/commits/176216373b289729496240bf05a0c5f7cb41a821